content/changelog/2025-02-28.md

```diff
@@ -0,0 +1,102 @@
+
+## Neon joins GitHub's Secret Scanning Partner Program 🔒
+
+We're excited to announce that Neon is now a GitHub Secret Scanning Partner. This GitHub program helps us protect Neon users by automatically detecting exposed Neon database credentials and API keys in public repositories and `npm` packages before they can be exploited.
+
+To learn more about how leaks are detected and how that's handled, please read: [Neon Joins GitHub's Secret Scanning Partner Program](tbd).
+
+## Scheduled updates for Business plan accounts
+
+A few weeks ago, we announced _scheduled updates_ for Neon, which include Postgres version upgrades, security patches, and Neon feature enhancements.
+
+Updates only take a few seconds and are applied at the scheduled time or the next time your compute restarts.
+
+Updates for Business plan accounts will start rolling out next week. You can check for updates notices and choose a preferred update window — [learn how](/docs/manage/updates#updates-on-paid-plans).
+
+![Paid plan updates UI](/docs/manage/paid_plan_updates.png)
+
+_Computes larger than 8 CU or configured to scale beyond 8 CU are not updated automatically._
+
+For more information about updates, see our [Updates documentation](/docs/manage/updates). If you have questions, please reach out to us on [Discord](https://discord.gg/92vNTzKDGp) or [contact Neon Support](https://console.neon.tech/app/projects?modal=support).
+
+## Early Access Program now available for organizations 🔓
+
+Organization admins can now enable Early Access for their entire organization. Once enabled, all organization members can preview upcoming Neon features across their organization's projects.
+
+![Early Access settings for organizations](/docs/relnotes/org_early_acces.png)
+
+Read more about the [Early Access Program](/docs/introduction/roadmap#join-the-neon-early-access-program).
+
+## Postgres version updates
+
+We updated supported Postgres versions to [14.17](https://www.postgresql.org/docs/release/14.17/), [15.12](https://www.postgresql.org/docs/release/15.12/), [16.8](https://www.postgresql.org/docs/release/16.8/), and [17.4](https://www.postgresql.org/docs/release/17.4/), respectively.
+
+When a new minor version is available on Neon, it is applied the next time your compute restarts (for any reason). For more about how we handle Postgres version upgrades, refer to our [Postgres version support policy](/docs/postgresql/postgres-version-policy).
+
+<details>
+
+<summary>**Fixes & improvements**</summary>
+
+- **Neon Console**
+
+  - Restricted Neon Auth installation and removal to organization admins only
+
+- **Drizzle Studio update**
+
+  We updated the Drizzle Studio integration that powers the **Tables** page in the Neon Console to version 1.0.15. For the latest improvements and fixes, see the [Neon Drizzle Studio Integration Changelog](https://github.com/neondatabase/neon-drizzle-studio-changelog/blob/main/CHANGELOG.md).
+
+- **API Updates**
```

Comment:
Hi, could you also include the changes in https://github.com/neondatabase/cloud/pull/24569? 

I believe we mentioned in previous changelogs that this change would come and I am unsure if we mentioned that buttons are now blocked less aggressively in the changelogs